benches/speed_tests.rs

```diff
@@ -396,10 +403,154 @@ pub fn idpf(c: &mut Criterion) {
     }
 }
 
+/// Benchmark Poplar1.
+#[cfg(feature = "experimental")]
+pub fn poplar1(c: &mut Criterion) {
+    let test_sizes = [16_usize, 128, 256];
+
+    let mut group = c.benchmark_group("poplar1_shard");
+    for size in test_sizes.iter() {
+        group.throughput(Throughput::Bytes(*size as u64 / 8));
+        group.bench_with_input(BenchmarkId::from_parameter(size), size, |b, &size| {
+            let bits = iter::repeat_with(random).take(size).collect::<Vec<bool>>();
+            let measurement = IdpfInput::from_bools(&bits);
+            let vdaf = Poplar1::new_aes128(bits.len());
+
+            b.iter(|| {
+                vdaf.shard(&measurement).unwrap();
+            });
+        });
+    }
+    drop(group);
+
+    let mut group = c.benchmark_group("poplar1_prepare_init");
+    for size in test_sizes.iter() {
+        group.throughput(Throughput::Bytes(*size as u64 / 8));
+        group.bench_with_input(BenchmarkId::from_parameter(size), size, |b, &size| {
+            let vdaf = Poplar1::new_aes128(size);
+            let verify_key = random();
+            let nonce: [u8; 16] = random();
+
+            // Parameters are chosen to match Chris Wood's experimental setup:
+            // https://github.com/chris-wood/heavy-hitter-comparison.
```

Comment:
```suggestion
            // https://github.com/chris-wood/heavy-hitter-comparison
```
Makes it easier to copy-paste the URL!